Dockerfile

```diff
@@ -1,13 +1,54 @@
+# syntax=docker/dockerfile:1.6@sha256:ac85f380a63b13dfcefa89046420e1781752bab202122f8f50032edf31be0021
```

Comment:
I am not sure about pinning this either for similar reasons, adding this now might mean that this won't budge at all in the next 3 years or so :(, so maybe `# syntax=docker/dockerfile:1` could be sufficient? Not greatly opposed that this stays pinned either as it is less work/important than apt pinning anyway.

Reply:
mhh okay no problem